Resources/Locale/en-US/communications/communications-console-component.ftl

```diff
@@ -46,3 +46,12 @@ comms-console-announcement-title-station = Communications Console
 comms-console-announcement-title-centcom = Central Command
 comms-console-announcement-title-nukie = Syndicate Nuclear Operative
 comms-console-announcement-title-station-ai = Station AI
+comms-console-announcement-title-wizard = Wizard
+
+# Funkystation
+comms-console-announcement-title-science = Science
+comms-console-announcement-title-cargo = Cargo
+comms-console-announcement-title-security = Security
+comms-console-announcement-title-medical = Medical
+comms-console-announcement-title-service = Service
+comms-console-announcement-title-engi = Engineering
```

Comment:
Move all of those into `OmuStation/Resources/Locale/en-US/_Omu/communications/communications-console-coponent.ftl`